hosted/gke/support_matrix/support_matrix_provisioning_test.go

```diff
@@ -32,11 +30,14 @@ var _ = Describe("SupportMatrixProvisioning", func() {
 			)
 			BeforeEach(func() {
 				clusterName = namegen.AppendRandomString("gkehostcluster")
-				pipeline.UpdateHostedKubernetesVField(provisioninginput.GoogleProviderName.String(), version)
 				var err error
 				gkeConfig := new(management.GKEClusterConfigSpec)
 				config.LoadAndUpdateConfig(gke.GKEClusterConfigConfigurationFileKey, gkeConfig, func() {
 					gkeConfig.ProjectID = project
+					gkeConfig.KubernetesVersion = &version
+					gkeConfig.Zone = zone
+					labels := helper.GetLabels()
```

Comment:
I tried doing that earlier but since we are running multiple tests in parallel, I have run into cases where one test is writing to the file and other test is reading from it and there is a conflict because test-2 reads test-1's data. This is particularly seen with AKS, where test-1 takes ResourceGroup of test-2 and then we run into error.

Also, we are using 2 different structs for provisioning and importing tests, so I cannot use the suite file to assign label to the config file since there is no nice way to identify if we are running Provisioning tests or importing.